core/common/src/main/java/alluxio/metrics/WorkerMetrics.java

```diff
@@ -0,0 +1,26 @@
+/*
+ * The Alluxio Open Foundation licenses this work under the Apache License, version 2.0
+ * (the "License"). You may not use this work except in compliance with the License, which is
+ * available at www.apache.org/licenses/LICENSE-2.0
+ *
+ * This software is distributed on an "AS IS" basis, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+ * either express or implied, as more fully set forth in the License.
+ *
+ * See the NOTICE file distributed with this work for information regarding copyright ownership.
+ */
+
+package alluxio.metrics;
+
+/**
+ * Metrics of an Alluxio worker.
+ */
+public final class WorkerMetrics {
```

Comment:
should we move all worker metrics to this class?

Reply:
That would bloat up this PR. I prefer to do the clean up separately. Note that this PR does not go to master, but a branch